fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/ConstantOperator.java

```diff
@@ -186,6 +186,26 @@ public static ConstantOperator createBinary(byte[] value, Type binaryType) {
         return new ConstantOperator(value, binaryType);
     }
 
+    public static ConstantOperator createExampleValueByType(Type type) {
```

Comment:
`createDefaultValueByType`?

Reply:
This is just for create a constant value to replcae a columnRefOperator. It's actualy not a defalut value for a type. I think defalut may cause some misunderstanding.